e2etest/bgp_test.go

```diff
@@ -464,18 +473,156 @@ var _ = ginkgo.Describe("BGP", func() {
 			),
 		)
 	})
+
+	ginkgo.Context("validate configuration changes", func() {
+		ginkgo.AfterEach(func() {
+			// Clean previous configuration.
+			err := updateConfigMap(cs, configFile{})
+			framework.ExpectNoError(err)
+		})
```

Comment:
Why is this necessary? The one defined in the beginning of the Describe isn't enough?

Reply:
That's an after each to we do a cleanup before the next test.

Reply:
but doesn't the first aftereach already takes care of that? https://github.com/metallb/metallb/blob/9a08426c6d753d0e9323dac60152ba8f123cb507/e2etest/bgp_test.go#L113

Reply:
It does

Reply:
right, i will remove it